### DISCUSSION TITLE
On reply a comment got a JS alert error "Unable to post your comment!, please try again"

### DISCUSSION THREAD

**Discussion by devmedtz** [Radi85/Comment#177] [Q&A]
I was set up Django commenting dab on my project by the following documentation, but I got a JS alert error when reply to a comment "Unable to post your comment! please try again" 

![Capture](https://user-images.githubusercontent.com/32010856/112450772-c7e7c280-8d65-11eb-885f-21d907815833.PNG)

Please help to resolve a problem

**Comment by HelloMukama** [comment]
I get this error as well but im going to dive into the code and figure it out.
Doesnt seem to come from the server side tho

**Reply by abhiabhi94** [reply]
this has been fixed in #178. You may clone the current code and use these [installation instructions](https://github.com/radi85/comment#installation-1) to use the latest version. 
Most probably there will be a release to the `pypi` this weekend, so you can use that as well if you can wait.
